zh_cn.lang

```diff
@@ -415,98 +415,119 @@ gui.xaero_category_enable_protection=保护：关
-gui.xaero_radar_render_radar_over_frame=在框架之上渲染雷达
+gui.xaero_chunk_grid_line_width=区块网格线宽度
+gui.xaero_radar_render_over_minimap=在小地图边框之上渲染
+gui.xaero_box_radar_render_over_minimap=在小地图边框上渲染实体，而不是在边框内渲染它们。 以这种方式渲染的实体的行为类似于路径点，并且在超出边界时会粘在边界上。 可见实体仍然仅限于游戏客户端加载的实体。 \n \n 选项“玩家列表” - 仅当按下“玩家列表”键绑定（默认为 TAB）时才在小地图上渲染。
+gui.xaero_radar_render_radar_over_frame=在边框之上渲染雷达
 gui.xaero_radar_over_map_never=从不
 gui.xaero_radar_over_map_list=玩家列表
 gui.xaero_radar_over_map_always=总是
-gui.xaero_compass_over_everything=指南针置于顶层
-gui.xaero_box_compass_over_everything=将指南针字母（N、E、S、W）渲染在所有地图元素上，而不是渲染在路径点等元素下方。
+gui.xaero_compass_over_everything=指南针在最上层
+gui.xaero_box_compass_over_everything=在所有地图元素上方渲染指南针文字（东、西、南、北），而不是在某些元素下方，例如路径点。
 gui.xaero_waypoint_server_shared=服务器
-gui.xaero_pac_players=显示派对/盟友玩家
-gui.xaero_box_pac_players=Display players whose locations are shared with you (by the Open Parties and Claims mod) when they are out of the minimap frame bounds or not visible for another reason.
-gui.xaero_toggle_pac_players=启用派对/盟友玩家
-gui.xaero_toggle_pac_chunk_claims=启用区块声明
+gui.xaero_waypoint_somebody_shared=某人
+gui.xaero_pac_players=展示帮派/盟友玩家
+gui.xaero_box_pac_players=当玩家超出小地图框体边界或因其他原因不可见时，显示与你共享位置的玩家（由 Open Parties and Claims 模组提供）。
+gui.xaero_toggle_pac_players=切换帮派/盟友玩家
+gui.xaero_toggle_pac_chunk_claims=切换区块认领
 gui.xaero_pac_marked_for_forceload=（可强制加载）
-gui.xaero_pac_claim_tooltip=%1$s的声明%2$s
-gui.xaero_pac_server_claim_tooltip=服务器声明%1$s
-gui.xaero_pac_expired_claim_tooltip=过期的声明%1$s
-gui.xaero_pac_claims=显示区块声明
-gui.xaero_box_pac_claims=用颜色叠加的方式显示来自Open Parties and Claims模组的区块声明。
-gui.xaero_pac_current_claim=显示当前声明
-gui.xaero_box_pac_current_claim=显示你所在的区块内的声明信息。区块声明需要Open Parties and Claims模组。
+gui.xaero_pac_claim_tooltip=%1$s 的认领 %2$s
+gui.xaero_pac_server_claim_tooltip=服务器认领 %1$s
+gui.xaero_pac_expired_claim_tooltip=到期的认领 %1$s
+gui.xaero_pac_claims=显示区块认领
+gui.xaero_box_pac_claims=将来自 Open Parties and Claims 模组的区块认领显示为彩色叠加层。
+gui.xaero_pac_claims_border_opacity=区块认领边界不透明度
+gui.xaero_box_pac_claims_border_opacity=区块认领叠加层的边缘不透明度。区块认领需要 Open Parties and Claims 模组。
+gui.xaero_pac_claims_fill_opacity=区块认领填充不透明度
+gui.xaero_box_pac_claims_fill_opacity=区块认领叠加层的填充颜色不透明度。区块认领需要 Open Parties and Claims 模组。
+gui.xaero_pac_current_claim=显示当前认领
+gui.xaero_box_pac_current_claim=显示有关你所在区块的认领信息。区块认领需要 Open Parties and Claims 模组。
+gui.xaero_category_include_list_include_in_super=包含在超类别中
+gui.xaero_box_category_include_list_include_in_super=自动将此列表中的所有实体以及此类别中包含的其他列表 包含在此类别的超类别/父级中，排除列表中的实体除外。 这样你就不必在每个级别上手动添加重复项。
+gui.xaero_temp_waypoints_global=临时路径点为全局
+gui.xaero_box_temp_waypoints_global=启用后，临时路径点将被视为全局路径点并忽略路径点渲染距离限制。
+gui.xaero_keep_enlarged_minimap_unlocked=放大时保持旋转
+gui.xaero_box_keep_enlarged_minimap_unlocked=如果可能的话，在放大时不要将小地图锁定在北方，这只适用于圆形小地图。
+gui.xaero_enlarged_minimap_a_toggle=切换放大小地图
```

Comment:
Not sure if this is correct. This option determines whether enlarged minimap is toggled or held. Please check.